### PR TITLE
Add aligned_to_consensus tool

### PR DESCRIPTION
**Pull request by ColineRoyaux**

### CODE REVIEW

File: ecology.yaml [hunk 1]
```diff
@@ -775,3 +775,7 @@ tools:
   - name: ab1_fastq_converter
     owner: ecology
     tool_panel_section_label: 'Convert Formats'
+    
+  - name: aligned_to_consensus
```
Comment (bgruening) [inline ecology.yaml:779]: Are you sure the tool is named like that in the TS? It is not available?

Comment (ColineRoyaux) [reply ecology.yaml]: It was right actually

**Comment by bgruening** [comment]
It will be installed over the weekend, let me know if you need it sooner.

**Comment by ColineRoyaux** [comment]
Thank you so much Bjorn ! It's not urgent don't worry :)

**Pull request merged**